src/SmartComponents/BaselinesPage/EditBaselinePage/EditBaseline/EditBaseline.js

```diff
@@ -45,10 +45,10 @@ class EditBaseline extends Component {
                 key='edit-baseline-table-header'
                 data-ouia-component-type='PF4/TableHeaderRow'
                 data-ouia-component-id='edit-baseline-table-header-row'>
-                { baselinesWrite ? <th></th> : null }
-                <th className="edit-baseline-header"><div>Fact</div></th>
-                <th className="edit-baseline-header"><div>Value</div></th>
-                <th></th>
+                { baselinesWrite ? <th className="sticky-header"></th> : null }
+                <th className="edit-baseline-header sticky-header"><div>Fact</div></th>
+                <th className="edit-baseline-header sticky-header"><div>Value</div></th>
+                <th className="sticky-header"></th>
```

Comment:
If anyone can help me figure out how to add a gray border to the bottom of this sticky header, that would be great. No matter where I add it, it doesn't show up. To view this table, navigate to Drift -> Baselines and click on a baseline to view its facts. You will see the gray border until the sticky header goes into affect. I can demo if you're having a hard time reproducing.

Reply:
Have you tried `!important`?

Reply:
@bastilian didn't seem to make a difference.